src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -303,13 +318,15 @@ def query_widgets
           ]
         end
 
+        # @macro seeAbstractWidget
         def store
           start_block, end_block = query_widgets
           len = end_block - start_block + 1
           bsize = @regions.first.block_size # where does this come from?
           @ptemplate.region = Y2Storage::Region.create(start_block, len, bsize)
         end
 
+        # @macro seeAbstractWidget
```

Comment:
@jreidinger What do you think about the current 90% required documentation coverage?
I do like meaningful documentation but currently it seems a big majority of it would be chasing macros that just tell YARD to shut up because "this is a in implementation of a well known API from a superclass" which YARD should be able to figure out by itself, if only it could do links across repos.